src/com/palmergames/bukkit/config/ConfigNodes.java

```diff
@@ -1648,6 +1648,11 @@
 			"# +------------------------------------------------------+ #",
 			"############################################################",
 			""),
+	INVITE_SYSTEM_INVITATION_SHOW_NATION_NAME(
+		"invite_system.invitation_show_nation_name",
+		"true",
+		"",
+		"# If you invite another player to your town, this will decide if it should show the nation name together with the town name, or just the town name."),

```

Comment:
This doesn't really need to be a config option, I think it's a good quality-of-life improvement on its own.

Reply:
Removed